src/gevent/monkey.py

```diff
@@ -1111,18 +1113,15 @@ def _get_script_help():
     modules = [x for x in patch_all_args if 'patch_' + x in globals()]
     script_help = """gevent.monkey - monkey patch the standard modules to use gevent.
 
-USAGE: ``python -m gevent.monkey [MONKEY OPTIONS] script [SCRIPT OPTIONS]``
+USAGE: ``python -m gevent.monkey [MONKEY OPTIONS] [--module] (script|module) [SCRIPT OPTIONS]``
 
 If no OPTIONS present, monkey patches all the modules it can patch.
-You can exclude a module with --no-module, e.g. --no-thread. You can
-specify a module to patch with --module, e.g. --socket. In the latter
+You can exclude a module with --no-<module>, e.g. --no-thread. You can
+specify a module to patch with --<module>, e.g. --socket. In the latter
 case only the modules specified on the command line will be patched.
 
-.. versionchanged:: 1.3b1
-    The *script* argument can now be any argument that can be passed to `runpy.run_path`,
-    just like the interpreter itself does, for example a package directory containing ``__main__.py``.
-    Previously it had to be the path to
-    a .py source file.
+The default behavior is to execute the script passed as argument. If you with
+to run a module instead, pass the `--module` argument before the module name.
```

Comment:
This needs a `.. versionchanged:: 1.5` block.

Reply:
Those blocks are cool to generate doc but they are quite ugly to render in the script help in your terminal…